packages/tools/cookbook/src/utils/account-key.ts

```diff
@@ -10,9 +6,15 @@ const HELP: string =
  * @return Account key
  */
 export function accountKey(account: string): string {
-  if (!namespacedAccountRegex.test(account)) {
-    console.info(HELP);
+  const regExp: RegExp = new RegExp(/^([a-z]+\:)([a-z0-9]{64})$/g);
+  const [, namespace, publicKey] = regExp.exec(account) || [];
+
+  if (!namespace || !publicKey) {
+    console.info(
+      `Account name ${account} doesn't follow the recommended convention: 'k:publicKey'`,
```

Comment:
(nit) 
Do we want the error message here to highlight that not just `k:` accounts are allowed? For example, `t:publicKey` would pass the regex check.